### ISSUE TITLE
Agenda not writing in android 7.1 lineage os

### ISSUE THREAD

**Issue by adamsson6** [39aldo39/DecSyncCC#10]
The problem is a bit weird because it was working before and suddently it's not working anymore...
The problem occurs with a tablet LG-V500.  When I add entries in the calendar from the phone, and reset the calendar in the tablet app, in this way the sync works.
But when I create an entry from the tablet, it appears in my agenda but does not sync. And if I reboot the tablet, the calendars in decsync are unchecked, and if I check them, the entries I created before does not appear.
I wiped cache and dalvik cache and reinstalled the app but I can't find where does the problem come from...

Here are some adb logs, may be it can be useful :

```

04-03 06:56:18.542 26533 26568 W aCalendar: unknown calendar type org.decsync.calendars DecSync Calendars lecec DecSync Calendars
04-03 06:56:18.546 26533 26568 W aCalendar: unknown calendar type org.decsync.calendars DecSync Calendars agenda DecSync Calendars
04-03 06:56:18.678 26533 26533 E PhoneWindow: Circular progress bar not located in current window decor
04-03 06:56:18.982 26154 26154 I CalendarProvider2: Sending notification intent: Intent { act=android.intent.action.PROVIDER_CHANGED dat=content://com.android.calendar flg=0x20000000 }
04-03 06:56:18.987 26154 26154 W ContentResolver: Failed to get type for: content://com.android.calendar (Unknown URL content://com.android.calendar)
04-03 06:56:19.012 26127 26127 D AlertReceiver: onReceive: a=android.intent.action.PROVIDER_CHANGED Intent { act=android.intent.action.PROVIDER_CHANGED dat=content://com.android.calendar flg=0x20000010 cmp=com.android.calendar/.alerts.AlertReceiver }
 
```

**Comment by 39aldo39** [comment]
It is indeed a bit weird. Looking at the logs, it looks like the calendar account cannot be created. Can you in the Settings app go to "Accounts" and check whether there is a account "DecSync Calendars"?

**Comment by adamsson6** [comment]
I checked, there is no "DecSync Calendar" account. Decsync Contacts is here, working and syncing but not calendar.

**Comment by 39aldo39** [comment]
Indeed, the calendar account cannot be created. DecSync CC tries to create the account every time you open the app, so it keeps failing. It is also strange that contacts accounts are successfully created. I don't really know how it can fail. Some guesses:
* Can you create calendar accounts with other apps?
* Have you granted DecSync CC all the necessary permissions?
